string source_topic
string destination_topic
uint32 queue_size

